### PR TITLE
Cryptocurrency and Sports update related API

### PR DESCRIPTION
**Pull request by abhijeet007rocks8**
APIs added are -:
- [API NBA](https://api-sports.io/documentation/nba/v2)
- [Brave New Coin](https://docs.bravenewcoin.com/)

Related to #2 @TheLearneer

### CODE REVIEW

File: src/content/apis/cryptocurrency/brave-new-coin.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Brave New Coin
+description: Providing data, indexes, analysis and insight on the cryptographic asset marketplace
+categories:
+  - cryptocurrency
+features:
+  https: true
+url: https://docs.bravenewcoin.com/
+logo: https://docs.bravenewcoin.com/images/logo.png
```
Comment (TheLearneer) [inline src/content/apis/cryptocurrency/brave-new-coin.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

File: src/content/apis/sports/api-nba.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: API NBA
+description: All NBA Stats DATA, Games, Livescore, Standings, Players, Statistics…
+categories:
+  - sports
+features:
+  https: true
+url: https://api-sports.io/documentation/nba/v2
+logo: https://www.api-basketball.com/public/img/home1/hero-banner1.png
```
Comment (TheLearneer) [inline src/content/apis/sports/api-nba.md:9]: The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Hello @abhijeet007rocks8 Thank you for your contribution.
However for the PR to be accepted, please go through the requested changes once 🙏

**Comment by abhijeet007rocks8** [comment]
@TheLearneer The requested changes have been made

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**